tools/property-extractor/parser.py

```
@@ -0,0 +1,229 @@
+from file_pair import FilePair
+from tree_sitter import Language
+from property_bag import PropertyBag
+from copy import deepcopy
+import itertools as it
+import re
+
+# Temp
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)